game/gui/hud.py

```diff
@@ -10,12 +10,15 @@
 
 HP_WIDTH = 150
 HP_HEIGHT = 20
+SCREEN_WIDTH = 1000
+SCREEN_HEIGHT = 650
```

Comment:
These constants are defined in `core`, you don't want to duplicate them. Just do `core.SCREEN_WIDTH` and `core.SCREEN_HEIGHT`.